apps/emqx_opentelemetry/src/emqx_otel_schema.erl

```diff
@@ -29,18 +29,18 @@ roots() -> ["opentelemetry"].
 
 fields("opentelemetry") ->
     [
-        {exporter,
+        {"exporter",
```

Comment:
keep using atoms for filed names

Reply:
Q: why is it better to use atoms for these?

Reply:
Changed back to atoms